app/tut/ga/creatlach.md

```diff
@@ -0,0 +1,967 @@
+---
+title: Creatlacha
+prev: peint
+prev-text: Am Péinteála
+tsn-fname: outline
+---
+
+# [[Oibiachtaí|Objects]]
+
+Go minic sa teagasc seo chonaiceamar an [[siombail|symbol]] `@`, anois tá sé in am caint faoi
+[[cad is brí leis|what it means]]. Baineadh úsáid as i [[sloinn|expressions]] cosúil le
+`ciorcal@stáitse`{.setanta} agus `slánuimh_rand@mata`{.setanta}. Chun na sloinn sin a thuiscint, is
+gá dúinn foghlaim faoi **[[oibiachtaí|objects]]**.
+
+## Cad iad Oibiachtaí?
+
+Is féidir leat smaoinigh faoi mar [[bailiúchán|collection]] [[luachanna|values]] agus
+[[iompraíochtaí|behaviours]]. Is féidir le oibiacht aon luach is mian linn a stóráil: uimhreacha,
+téacs, gníomhartha srl. Tá ainm [[bainte|associated]] le gach [[ball|member]] den oibiacht, díreach
+mar a bhíonn ainmneacha bainte le [[athróga|variables]].
+
+Mar shampla, ba féidir linn oibiacht a bheith againn a [[léiríonn duine éigin|represents a person]]:
```

Comment:
Níl fhios agam an é féad an briathar is fearr anseo, ach seo mar a úsáidtear é https://www.teanglann.ie/ga/gram/f%c3%a9ad

B'fhearr liom féin rud éigin fearacht
Abair go raibh oibiacht againn a léiríonn duine éigin. Bheadh aois, seoladh nó ainm mar bhaill ag an oibiacht, ag stóráil aois, seoladh agus ainm an duine faoi seach.

Reply:
Athraithe